SalesforceCommerceCloud/pwa-kit
# Generator fix siteId to use RefArch instead of RefArchGlobal

# Description



The generator creates a project using the single-locale `en-US`. 
The `en-US` locale only exists in the `siteId` `RefArch` in our demo ODS.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Use siteId `RefArch` instead of `RefArchGlobal`.

# How to Test-Drive This PR

- Run the generator locally 
```
node scripts/create-mobify-app-dev.js
```
- Choose to generate the project `retail-react-app-demo`.
- Start the server on the generated project `npm start`.
- Smoke test the App and verify that there are not errors related to the en-US locale http://localhost:3000/

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
